### ISSUE TITLE
Slack RTM Driver Broken

### ISSUE THREAD

**Issue by Alexander-Karsten** [botman/botman#531]
- BotMan Version: 2.0.3
- PHP Version: 7.1
- Messaging Service(s): Slack
- Cache Driver: None

### Description:
Continued from https://github.com/botman/botman/issues/523

The Slack RTM driver currently does not work, if you run the example code provided in the docs you will be presented with the following error:

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to BotMan\Drivers\Slack\SlackRTMDriver::__construct() must be of the type array, object given, called in /vagrant/N800/app/vendor/botman/botman/src/Drivers/DriverManager.php on line 148 and defined in /vagrant/N800/app/vendor/botman/driver-slack/src/SlackRTMDriver.php:47
```

This appears to be because `DriverManager::verifyServices` is calling the Driver with incorrect parameters. Amending line 457 (`return true;`) results in this working.

### Steps To Reproduce:
1. Install following dependencies:
```
    "botman/botman": "^2.0",
    "mpociot/slack-client": "^0.3",
    "botman/driver-slack": "^1.0"
```
2. Use the example code provided on the website
3. Get above error
4. 😞

**Comment by christophrumpel** [comment]
Hey @Alexander-Karsten , thx for mentioning. We found the issue and are working on a fix.

**Comment by mpociot** [comment]
Hi @Alexander-Karsten ,

the issue is fixed with BotMan 2.0.4

**Comment by Alexander-Karsten** [comment]
Can confirm everything is working beautifully. Thanks a lot for your help as well as providing this great little tool @christophrumpel and @mpociot. :)
